frontend/src/pages/auth/Redirection.tsx

```diff
@@ -0,0 +1,63 @@
+import { useContext, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { AuthContext } from '@hooks/context/auth';
+
+import { getCookieToken, setCookieToken } from '@utils/cookie';
+import { getFetch } from '@utils/fetch';
+
+interface AuthResponse {
+  accessToken: string;
+}
+
+const getAuthInfo = async (url: string): Promise<AuthResponse> => {
+  return await getFetch<AuthResponse>(url);
+};
+
+export default function Redirection() {
+  const { loggedInfo, setLoggedInfo } = useContext(AuthContext);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    (async () => {
```

Comment:
혹시 즉시실행함수를 사용하신 이유가 있나요?

Reply:
해당 코드가 바로 시작되었으면 좋겠는데 즉시실행함수로 감싸지 않으면 오류가 납니다!

Reply:
```
'() => Promise<void>' 형식의 인수는 'EffectCallback' 형식의 매개 변수에 할당될 수 없습니다.
  'Promise<void>' 형식은 'void | Destructor' 형식에 할당할 수 없습니다.ts(2345)
Effect callbacks are synchronous to prevent race conditions. Put the async function inside:

useEffect(() => {
  async function fetchData() {
    // You can await here
    const response = await MyAPI.getData(someId);
    // ...
  }
  fetchData();
}, [someId]); // Or [] if effect doesn't need props or state
```